packages/core/utils/src/config.js

```diff
@@ -16,56 +16,24 @@ export type ConfigOptions = {|
   parse?: boolean,
 |};
 
-export async function resolveConfig(
+export function resolveConfig(
   fs: FileSystem,
   filepath: FilePath,
   filenames: Array<FilePath>,
-  opts: ?ConfigOptions,
-  root: FilePath = path.parse(filepath).root,
-): Promise<FilePath | null> {
-  filepath = await fs.realpath(path.dirname(filepath));
-
-  // Don't traverse above the module root
-  if (path.basename(filepath) === 'node_modules') {
-    return null;
-  }
-
-  for (const filename of filenames) {
-    let file = path.join(filepath, filename);
-    if ((await fs.exists(file)) && (await fs.stat(file)).isFile()) {
-      return file;
-    }
-  }
-
-  if (filepath === root) {
-    return null;
-  }
-
-  return resolveConfig(fs, filepath, filenames, opts);
+): Promise<?FilePath> {
+  // TODO: realpath
```

Comment:
Unclear about this: it doesn't break any tests at the moment to remove it. However, I think it will change again soon anyway though perhaps slightly different (realpath the result rather than the input). See #5183.

Reply:
I believe this will be unnecessary once #5508 is merged since all asset file paths (and therefore where we are searching for config from) will be realpaths. Remove the TODO comment in that PR.